src/render-html/generatedEs3.js

```diff
@@ -178,16 +178,26 @@ var handleMessageTyping = function handleMessageTyping(msg) {
 
 var handleLongPress = function handleLongPress(e) {
   if (!lastTouchEventTimestamp || Date.now() - lastTouchEventTimestamp < 500) return;
-
   lastTouchEventTimestamp = 0;
-
   sendMessage({
     type: 'longPress',
     target: e.target.matches('.header') ? 'header' : 'message',
     messageId: +getMessageIdFromNode(e.target)
   });
 };
 
+var handleImageLongPress = function handleImageLongPress(e) {
```

Comment:
Much of this function duplicates what's in `handleLongPress`. Let's avoid the duplication, please -- instead, add a conditional to `handleLongPress`.